File: ADS 2023.2/AlgoritimosProgramacao/Algoritmos/ifpi-ads-2023.2/modulo2-modularizacao/area_triangulo.py
def main():
    print('***** AREA TRIANGULO *****')
    
    # Entrada
    base = float(input("Base: "))
    altura = float(input("Altura: "))
    
    # Processamento 
    area = calcular_area(base, altura)

    # Saída
    print(f"A área do triângulo é: {area}")



# Definição de uma função em python
def calcular_area(b, a):
    resultado = (b * a) / 2
    return resultado

# Chamar a função main
main()

File: ADS 2023.2/AlgoritimosProgramacao/Algoritmos/ifpi-ads-2023.2/modulo2-modularizacao/area_triangulo.js
import { question } from "readline-sync";

function main(){

    console.log('***** AREA Triangulo *****')
    // Entrada
    const base = Number(question('Base: '))
    const altura = Number(question('Alutura: '))

    // Processamento
    const area = calcular_area(base, altura)

    // Saída
    console.log(`A área do triângulo é: ${area}`)



}
    // definição de uma função
    function calcular_area(b, a){
        const resultado = (b * a)/2
        return resultado
    }
  
main()

